triggers.gs

```diff
@@ -0,0 +1,77 @@
+/**
+* Trigger for thr rule for the number of executions for the script
+*/
+function mostExecutedTrigger() {
+  var files = DriveApp.getFilesByName("Admin Rules For Apps Scripts");
```

Comment:
What is this hard coded script. will it be the same for everyone who installs your add on?

Reply:
It will be same for everyone
Updated variables in the constant file